src/Bedrock.Framework.Experimental/Protocols/Amqp/AmqpProtocolversionHeader.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Bedrock.Framework.Infrastructure;
+using System;
+using System.Buffers;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Bedrock.Framework.Experimental.Protocols.Amqp
+{
+    public class AmqpProtocolversionHeader : IAmqpMessage
+    {
+        private ReadOnlySpan<byte> protocolHeader => new byte[] { (byte)'A', (byte)'M', (byte)'Q', (byte)'P', 0, 0, 9, 1 };
```

Comment:
```suggestion
        private ReadOnlySpan<byte> ProtocolHeader => new byte[] { (byte)'A', (byte)'M', (byte)'Q', (byte)'P', 0, 0, 9, 1 };
```
as it's a property. 

Note: you can also use it as a local variable in `Write`-method -- the ROS-trick still applies.

Reply:
in previous version, it was a local variable, but i'll need later for TryParse method, so i moved it as a property